x-pack/plugins/integration_assistant/public/common/lib/api_parsers.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { InstallPackageResponse } from '../../../common/types';
+
+/**
+ * This is a hacky way to get the integration name from the response.
+ * Since the integration name is not returned in the response we have to parse it from the ingest pipeline name.
+ * TODO: Return the package name from the fleet API.
+ */
+export const getIntegrationNameFromResponse = (response: InstallPackageResponse) => {
```

Comment:
May be add a simple test for success and failure scenario , just to make sure the hack works as expected?